### PR TITLE
Update id_ID

### PR DESCRIPTION
**Pull request by mohammadrizki**
- Update translation
- Change word choices
- Fix typos

### CODE REVIEW

File: locale/id_ID/author.xml [hunk 1]
```diff
@@ -1,122 +1,118 @@
...
+	<message key="author.submit.reorderInstructions">Mengurutkan nama penulis agar ditampilkan seperti urutan dalam terbitan.</message> 
+	<message key="author.submit.reorder">Mengurutkan nama penulis</message> 
+	<message key="author.submit.selectPrincipalContact">Kontak Utama untuk Korespondensi Editorial.</message>	
+	<message key="author.submit.addAuthor">Tambah Penulis</message> 
+	<message key="author.submit.deleteAuthor">Hapus Penulis</message> 
+	<message key="author.submit.submissionIndexingDescription">Masukkan istilah untuk mengindeks naskah; pisahkan istilah dengan titik koma (term1; term2; term3).</message> 
+	<message key="author.submit.coverageInstructions">Mengacu pada lokasi geo-spasial, liputan atau cakupan kronologis atau historis, dan/ atau karakteristik sampel penelitian.</message> 
+	<message key="author.submit.languageInstructions"><![CDATA[English=en; French=fr; Spanish=es. <a href="http://www.loc.gov/standards/iso639-2/php/code_list.php" target="_blank">Kode Tambahan</a>.]]></message>	<message key="author.submit.form.authorRequired">Setidaknya dibutuhkan satu penulis.</message>
```
Comment (mtub) [inline locale/id_ID/author.xml]: Are you sure you want to leave this English string in here? If someone is looking for missing keys in a translation, they will not discover this…

File: locale/id_ID/submission.xml [hunk 2]
```diff
@@ -1,78 +1,79 @@
...
+	<message key="submission.event.publicIdentifiers">Identifier publik naskah telah diperbaharui.</message>
+
+	<!-- galleys grid -->
+	<message key="grid.issueEntry.availableGalley.title">Persetujuan Format</message>
+	<message key="grid.issueEntry.availableGalley.removeMessage"><![CDATA[<p>Galley ini <em>tidak tersedia lagi</em> untuk pembaca.</p>]]></message>
+	<message key="grid.issueEntry.availableGalley.message"><![CDATA[<p>Galley ini <em>akan tersedia</em> untuk pembaca.</p>]]></message>
+	<message key="grid.action.availableArticleGalley">Aktifkan galley ini</message>
+
+	<!-- galley files grid -->
+	<message key="submission.galleyFiles">File Galley</message>
```
Comment (mtub) [inline locale/id_ID/submission.xml]: This looks like English text again. Are you sure this is correct?

File: locale/id_ID/submission.xml [hunk 3]
```diff
@@ -1,78 +1,79 @@
...
+	<message key="submission.proofReadingDescription"><![CDATA[Layout editor mengunggah file siap terbit.  Gunakan <em>Tugaskan Auditor</em> untuk menentukan penulis atau pengguna lainnya untuk melakukan proofreading terhadap galley, dan mengunggah file yang telah dikoreksi sebelum publikasi dilakukan.]]></message>
+	<message key="grid.action.approveProof">Setujui proof ini untuk dimasukkan dalam galley.</message>
+	<message key="grid.action.pageProofApproved">Proof ini telah disetujui untuk dimasukkan dalam galley.</message>
+	<message key="submission.submit.titleAndSummary">Judul dan Abstrak</message>
+	<message key="submission.submit.upload.description">Unggah file terkait naskah ini, termasuk artikel, multimedia, set data, artwork, dll.</message>
+	<message key="submission.pageProofs">Proofreading</message>
```
Comment (mtub) [inline locale/id_ID/submission.xml]: English text…

**Comment by mtub** [comment]
Are you aware of the work @awangga  is doing? See #1258…

**Comment by mohammadrizki** [comment]
Thanks @mtub, in Indonesia we do use English words for those technical
terms (file, galley, proofread, copyedit). People working in publication
field will readily understand those words. And, yes, I am aware off Awangga
work.
Regards,

Mohammad

On Feb 23, 2017 4:32 AM, "Marco Tullney" <notifications@github.com> wrote:

*@mtub* commented on this pull request.

Thank you, @mohammadrizki <https://github.com/mohammadrizki> . Can you have
a look at my comments?
------------------------------

In locale/id_ID/author.xml
<https://github.com/pkp/ojs/pull/1272#discussion_r102562031>:

> +	<message key="author.submit.verifyChecklist">Anda harus memastikan semua artikel tercantum dalam daftar nama naskah yang memuaskan sebelum dilanjutkan.</message>	
+	<message key="author.submit.copyrightNoticeAgree">Penulis setuju
dengan persyaratan hak cipta, yang akan berlaku pada naskah ini saat
diterbitkan oleh jurnal (komentar untuk editor dapat ditambahkan di
bawah).</message>
+	<message key="author.submit.copyrightNoticeAgreeRequired">Anda harus
menyetujui persyaratan hak cipta sebelum melanjutkan</message>
+	<message key="submission.submit.coverNote">Komentar untuk Editor</message>
+	<message key="author.submit.comments">Masukkan Teks (pilihan)</message>
+	<message key="author.submit.form.sectionRequired">Pilih bagian
jurnal yang sesuai untuk penyerahan ini.</message>
+	<message key="author.submit.form.localeRequired">Pilih bahasa
penyerahan naskah.</message>
+	<message key="submission.submit.privacyStatement">Pernyataan Privasi
Jurnal</message>
+	<message key="author.submit.reorderInstructions">Mengurutkan nama
penulis agar ditampilkan seperti urutan dalam terbitan.</message>
+	<message key="author.submit.reorder">Mengurutkan nama penulis</message>
+	<message key="author.submit.selectPrincipalContact">Kontak Utama
untuk Korespondensi Editorial.</message>
+	<message key="author.submit.addAuthor">Tambah Penulis</message>
+	<message key="author.submit.deleteAuthor">Hapus Penulis</message>
+	<message key="author.submit.submissionIndexingDescription">Masukkan
istilah untuk mengindeks naskah; pisahkan istilah dengan titik koma
(term1; term2; term3).</message>
+	<message key="author.submit.coverageInstructions">Mengacu pada
lokasi geo-spasial, liputan atau cakupan kronologis atau historis,
dan/ atau karakteristik sampel penelitian.</message>
+	<message key="author.submit.languageInstructions"><![CDATA[English=en;
French=fr; Spanish=es. <a
href="http://www.loc.gov/standards/iso639-2/php/code_list.php"
target="_blank">Kode Tambahan</a>.]]></message>	<message
key="author.submit.form.authorRequired">Setidaknya dibutuhkan satu
penulis.</message>

Are you sure you want to leave this English string in here? If someone is
looking for missing keys in a translation, they will not discover this…
------------------------------

In locale/id_ID/submission.xml
<https://github.com/pkp/ojs/pull/1272#discussion_r102564998>:

> +	<message key="submission.issueEntry.publicationMetadata">Publikasi</message>
+	<message key="submission.issueEntry.confirm">Buat entri terbitan
untuk naskah ini berdasarkan metadata berikut.</message>
+
+	
+	<message key="submission.event.articleGalleyMadeAvailable">Galley
artikel "{$galleyFormatName}" tersedia.</message>
+	<message key="submission.event.articleGalleyMadeUnavailable">Galley
artikel "{$galleyFormatName}" tersedia.</message>
+	<message key="submission.event.publicIdentifiers">Identifier publik
naskah telah diperbaharui.</message>
+
+	
+	<message key="grid.issueEntry.availableGalley.title">Persetujuan
Format</message>
+	<message key="grid.issueEntry.availableGalley.removeMessage"><![CDATA[<p>Galley
ini <em>tidak tersedia lagi</em> untuk pembaca.</p>]]></message>
+	<message key="grid.issueEntry.availableGalley.message"><![CDATA[<p>Galley
ini <em>akan tersedia</em> untuk pembaca.</p>]]></message>
+	<message key="grid.action.availableArticleGalley">Aktifkan galley
ini</message>
+
+	
+	<message key="submission.galleyFiles">File Galley</message>

This looks like English text again. Are you sure this is correct?
------------------------------

In locale/id_ID/submission.xml
<https://github.com/pkp/ojs/pull/1272#discussion_r102565033>:

> +	<message key="submission.event.publicIdentifiers">Identifier publik naskah telah diperbaharui.</message>
+
+	
+	<message key="grid.issueEntry.availableGalley.title">Persetujuan
Format</message>
+	<message key="grid.issueEntry.availableGalley.removeMessage"><![CDATA[<p>Galley
ini <em>tidak tersedia lagi</em> untuk pembaca.</p>]]></message>
+	<message key="grid.issueEntry.availableGalley.message"><![CDATA[<p>Galley
ini <em>akan tersedia</em> untuk pembaca.</p>]]></message>
+	<message key="grid.action.availableArticleGalley">Aktifkan galley
ini</message>
+
+	
+	<message key="submission.galleyFiles">File Galley</message>
+	<message key="submission.proofReadingDescription"><![CDATA[Layout
editor mengunggah file siap terbit.  Gunakan <em>Tugaskan Auditor</em>
untuk menentukan penulis atau pengguna lainnya untuk melakukan
proofreading terhadap galley, dan mengunggah file yang telah dikoreksi
sebelum publikasi dilakukan.]]></message>
+	<message key="grid.action.approveProof">Setujui proof ini untuk
dimasukkan dalam galley.</message>
+	<message key="grid.action.pageProofApproved">Proof ini telah
disetujui untuk dimasukkan dalam galley.</message>
+	<message key="submission.submit.titleAndSummary">Judul dan Abstrak</message>
+	<message key="submission.submit.upload.description">Unggah file
terkait naskah ini, termasuk artikel, multimedia, set data, artwork,
dll.</message>
+	<message key="submission.pageProofs">Proofreading</message>

English text…

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/pkp/ojs/pull/1272#pullrequestreview-23321312>, or mute
the thread

**Comment by mtub** [comment]
Thanks, @mohammadrizki. Your contribution makes OJS better. Will merge your changes now. 

Had to fix a few XML errors, some of which were a little difficult to track.

**Comment by mohammadrizki** [comment]
Aw, so sorry, my bad, @mtub, my bad. I forgot to do xml sanity check before
upload. Currently I am working on the lib/pkp/locale/id_ID, hopefully
finished next week.

On Feb 23, 2017 5:04 PM, "Marco Tullney" <notifications@github.com> wrote:

Thanks, @mohammadrizki <https://github.com/mohammadrizki>. Your
contribution makes OJS better. Will merge your changes now.

Had to fix a few XML errors, some of which were a little difficult to track.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/pkp/ojs/pull/1272#issuecomment-281934904>, or mute the
thread

**Pull request merged**